package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "calendar",
-  "version": "1.0.23",
+  "version": "1.0.24",
```

Comment:
Please don't update the package version, we increase it only when publishing a new version of the application at the end of the sprint.